commit 3ff5e1ec55c26d7dff5a5eef2cfa1666ed9e1185 (refs/changes/87/3287/1)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2018-02-01T18:26:31+00:00 (1 year, 8 months ago)
    
    Revert "OS-6582: libfakekernel changes break SmartOS platform build"
    
    This reverts commit b38c273fc5f09fcd363585faa4db621674a1850c.

diff --git a/nss-nspr/Makefile b/nss-nspr/Makefile
index 69e6c2e..b006810 100644
--- a/nss-nspr/Makefile
+++ b/nss-nspr/Makefile
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2018 Joyent, Inc.
+# Copyright 2016 Joyent, Inc.
 #
 
 VER =	nss-3.25
@@ -49,10 +49,3 @@ all_64: $(VER.64)/$(UNPACK_SENTINEL)
 install: all
 	DESTDIR=$(DESTDIR) ksh93 ./install-nss $(VER.32)
 	DESTDIR=$(DESTDIR) MACH64=amd64 ksh93 ./install-nss-64 $(VER.64)
-	# Apology in advance; see OS-6582 for why we're doing this.
-	if [[ ! -f $(DESTDIR)/usr/include/zconf.h.orig ]]; then \
-		cp $(DESTDIR)/usr/include/zconf.h \
-		   $(DESTDIR)/usr/include/zconf.h.orig; \
-		$(GPATCH) -d $(DESTDIR)/usr/include -p$(PATCHSTRIP) \
-		    <zconf.patch; \
-	fi
diff --git a/nss-nspr/zconf.patch b/nss-nspr/zconf.patch
deleted file mode 100644
index c759e97..0000000
--- a/nss-nspr/zconf.patch
+++ /dev/null
@@ -1,21 +0,0 @@
---- a/zconf.h	Wed Jan 31 18:16:08 2018
-+++ b/zconf.h	Wed Jan 31 18:17:11 2018
-@@ -8,6 +8,8 @@
- #ifndef ZCONF_H
- #define ZCONF_H
- 
-+#include <sys/machelf.h>
-+
- /*
-  * If you *really* need a unique prefix for all types and library functions,
-  * compile with -DZ_PREFIX. The "standard" zlib should be compiled without it.
-@@ -257,9 +259,6 @@
- #  define FAR
- #endif
- 
--#if !defined(__MACTYPES__)
--typedef unsigned char  Byte;  /* 8 bits */
--#endif
- typedef unsigned int   uInt;  /* 16 bits or more */
- typedef unsigned long  uLong; /* 32 bits or more */
- 
